samples/helm-deploy/README.md

```diff
@@ -0,0 +1,162 @@
+# How-To: Deploy to Kubernetes using Helm
+
+## Setup Dev Environment (optional)
+
+The easiest way to setup your environment is to install [Kind](https://kind.sigs.k8s.io/) or [Minikube](https://minikube.sigs.k8s.io/docs/start/). It is also valid to setup a cloud provider kubernetes cluster (i.e. AKS, EKS, GKE, ...)
+
+## Push an image to the Docker registry
+
+The following steps illustrates how to push a webservice image in order to be
+deployed in Kubernetes clsuter.
+
+1. Build an image using the following
+   [Dockerfile](.../../../../src/CarbonAware.WebApi/src/Dockerfile)
+
+   ```sh
+   cd <Dockerfile path>
+   docker build -t <registry>/myapp:v1 .
+   ```
+
+1. Login into docker using the username and password credentials that are needed in
+   order to push.
+
+   ```sh
+   docker login 
+   ```
+
+1. Push to Docker registry 
+
+   ```sh
+   docker push <registry>/myapp:v1
+   ```
```

Comment:
This section is not needed because default values.yaml refers container image from GitHub Container Registry.